jupyter/demo.ipynb

```diff
@@ -92,15 +149,23 @@
  ],
  "metadata": {
   "interpreter": {
-   "hash": "7a42a518fa29c240d94160a104e2571f110fb503155511d5e924fad0a3805a00"
+   "hash": "b23a54a307c68e72ca2a23e7951a8039dbbb834772c78cac9c7ee54b21666c94"
   },
   "kernelspec": {
-   "display_name": "Python 3.9.5 64-bit ('adaptivefiltering': conda)",
+   "display_name": "Python 3.9.4 64-bit ('filteradapt': conda)",
    "name": "python3"
   },
   "language_info": {
+   "codemirror_mode": {
+    "name": "ipython",
+    "version": 3
+   },
+   "file_extension": ".py",
+   "mimetype": "text/x-python",
    "name": "python",
-   "version": ""
+   "nbconvert_exporter": "python",
+   "pygments_lexer": "ipython3",
+   "version": "3.9.4"
```

Comment:
I am bit confused why `nbstripout` did not remove all of this metadata

Reply:
I have no idea